### DISCUSSION TITLE
How can I enter bootloader mode ?

### DISCUSSION THREAD

**Discussion by joygram** [irfanjmdn/r65#5] [Q&A] [answered]
my RK R65 not via supported keyboard. 
how can I enter boot loader mode ?

**Comment by BabyJenn** [comment]
> my RK R65 not via supported keyboard. how can I enter boot loader mode ?

FN+RIGHT SHIFT+ESC

**Comment by irfanjmdn** [comment, marked as answer]
Sorry for the insanely late response, there should be a button below the spacebar, on the right side, that you can press (refer to image). Hold the button with a tweezer/pen while connecting your cable to the computer. That should get you into bootloader mode.

![image](https://github.com/user-attachments/assets/2d15928e-4b28-4d4e-b7eb-6c9ae1af47e2)
